osu.Game/Online/Chat/Message.cs

```diff
@@ -63,5 +63,7 @@ public int CompareTo(Message other)
 
         // ReSharper disable once ImpureMethodCallOnReadonlyValueField
         public override int GetHashCode() => Id.GetHashCode();
+
+        public override string ToString() => $"[{ChannelId}] ({Id}) {Sender}: {Content}";
```

Comment:
Perhaps [`[DebuggerDisplay]`](https://docs.microsoft.com/pl-pl/visualstudio/debugger/using-the-debuggerdisplay-attribute?view=vs-2019) would be preferable for debug printing.

Reply:
I would have to regress my previous PR then, because it seems to fire off error messages with this class name without the current change.

Reply:
I'm not sure I understand.

Reply:
Right now:
read state error: "Failed to mark channel up to 'osu.Game.Online.Chat.Message' as read"
debugger: osu.Game.Online.Chat.Message

Your suggestion:
read state error: "Failed to mark channel up to 'osu.Game.Online.Chat.Message' as read"
debugger: "[0] (1) Craftplacer: This doesn't seem right"

If the current code was being pushed:
read state error: "Failed to mark channel up to '[0] (1) Craftplacer: This doesn't seem right' as read"
debugger: "[0] (1) Craftplacer: This doesn't seem right"

Reply:
Well, this doesn't mean that this *regresses* anything, but more that the previous logging code was *broken* in the first place. But fine, I suppose this needs to stay then.

Reply:
Well, regression is still a fancy word I didn't use yet, only heard from the dev team.